### DISCUSSION TITLE
"Metastore Migration Failed" Error after updating versions without rebooting

### DISCUSSION THREAD

**Discussion by npezzino** [datalust/seq-tickets#2186] [Help]
Our version of Seq was a 2024 version, it was not running properly due to legacy data. We rolled back to a 2022.1 version, and that was working properly. Once upgrading back to a 2024 version, it stopped working (we did not reboot). Rolling back to the working 2022.1 version also is throwing an error "Metastore Migration Failed". We did not reboot yet. 

`"Application startup exception","@l":"Fatal","@x":"Autofac.Core.DependencyResolutionException: An exception was thrown while activating λ:Seq.Cluster.IClusterNode -> λ:Seq.Server.StorageSubsystem.\r\n ---> System.Exception: Metastore migration failed.`

**Comment by nblumhardt** [comment]
Hi! Thanks for reaching out.

We'll need the full stack trace to be able to diagnose what's going on - could you please post as much of the start-up output as possible?

If you have a supported subscription, please also let us know (mailing `support@datalust.co` would be great).

Thanks!

**Reply by npezzino** [reply]
[seq-20240429.log](https://github.com/datalust/seq-tickets/files/15163505/seq-20240429.log)


I have attached the entire log from yesterday.
